utils/apiLog.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * /utils/apiLog.js
```

Comment:
There is a file activityLog in utils. Do you think it is better to move apiLog to activityLog because api request is a one of the activity type (api, worker and realtime)? Feel free to modify function names if conflicting.